include/BasicTypes.hpp

```diff
@@ -844,13 +845,13 @@ class NiBlockRefArray : public NiRefArray {
 
 	void GetIndices(std::vector<uint32_t>& indices) override {
 		for (auto& r : refs)
-			if (!r.IsEmpty())
+			if (!r.IsEmpty()) // is this a good idea?
 				indices.push_back(r.index);
```

Comment:
removed as part of sync

Reply:
I've removed the other empty checks not in ousnius, can open a PR if you want.